### PR TITLE
Fix error handling

### PR DESCRIPTION
**Pull request by krdurrett**
____ Wrote Tests __x__ Implemented ____ Reviewed

## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [] New feature
    [] Bug Fix
    [x] Refactor

## Description of Change:

    description: Get rid of error context and put logic in to components to render error modal
    
## Requested feedback: 
    
    I'd like feedback on...n/a

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [] My code has no unused/commented out code
    [] My code has no binding.pry's
    [] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

File: cypress/integration/allsightings-test.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ describe('All sightings page cypress test', () => {
     cy.get('.sightings-button').click()
     cy.get('\#\\32').click()
     cy.get('.sighting-photo')
-    cy.get('.sighting-title').contains('Section A-17 Prototype Labs')
+    cy.get('.sighting-title').contains('Sector B Coolant Reserve')
```
Comment (RosaTheDev) [inline cypress/integration/allsightings-test.js:30]: Were the cards switched up here?

**Review by RosaTheDev — APPROVED** [review]

File: cypress/integration/allsightings-test.js (hunk 1 above)
Comment (GraceGardner) [reply cypress/integration/allsightings-test.js:30]: @RosaTheDev They totally were!

File: src/App.js [hunk 2]
```diff
@@ -8,13 +8,11 @@ import SingleSighting from './Components/SingleSighting'
 import NavBar from './Components/NavBar';
 import ErrorModal from './Components/ErrorModal'
 import { NavigationProvider } from './Context/NavigationContext';
-import { ErrorProvider } from './Context/ErrorContext'
```
Comment (GraceGardner) [inline src/App.js:11]: Heck yeah! Deleting code!

**Review by jasmyn2244 — APPROVED** [review]
Great work!

File: src/Components/AllSightings.js [hunk 3]
```diff
@@ -52,11 +51,11 @@ const AllSightings = () => {
   }
 
   if (loading) return <Loading/>
+  
+  if (error) return <ErrorModal gqlError={error}/>
```
Comment (GraceGardner) [inline src/Components/AllSightings.js:55]: We have errors! Heck yeah!

**Review by GraceGardner — APPROVED** [review]
Nicely done Kayla! It looks great!

**Pull request merged**